desktop/main.js

```diff
@@ -100,23 +105,23 @@ const manuallyCheckForUpdates = (menuItem, browserWindow) => {
             if (downloadPromise) {
                 dialog.showMessageBox(browserWindow, {
                     type: 'info',
-                    message: 'Update Available',
-                    detail: 'The new version will be available shortly. We’ll notify you when we’re ready to update.',
-                    buttons: ['Sounds good'],
+                    message: Localize.translate(preferredLocale, 'checkForUpdatesModal.available.title'),
```

Comment:
I'd recommend saving these changes for another PR and keeping this one focused on the application menu to make it easier to test.

Reply:
I don't mind it leaving here, at this point moving it is going to be more work than testing it.